catroid/src/main/java/org/catrobat/catroid/formulaeditor/Sensors.java

```diff
@@ -41,7 +41,8 @@
 	DRONE_RECORDING, DRONE_NUM_FRAMES, COLLIDES_WITH_EDGE(true), COLLIDES_WITH_FINGER(true), OBJECT_X_VELOCITY(true),
 	OBJECT_Y_VELOCITY(true), OBJECT_ANGULAR_VELOCITY(true),
 	LAST_FINGER_INDEX, FINGER_X, FINGER_Y, FINGER_TOUCHED, NUMBER_CURRENT_TOUCHES, OBJECT_LOOK_NUMBER(true), OBJECT_LOOK_NAME(true),
```

Comment:
```suggestion
	LAST_FINGER_INDEX, FINGER_X, FINGER_Y, FINGER_TOUCHED, NUMBER_CURRENT_TOUCHES, 
```

Reply:
Please put the different sensors in separate lines (see also next suggestion in line 44 and 45).